test-suite/subperiodcoupons.cpp

```diff
@@ -281,8 +285,8 @@ void SubPeriodsCouponTest::testRegularSinglePeriodCouponAfterFixing() {
 
     Spread spread = 0.001;
     // For a single sub-period averaging method should not matter.
-    testSinglePeriodCouponReplication(start, end, spread, RateAveraging::Compound);
-    testSinglePeriodCouponReplication(start, end, spread, RateAveraging::Simple);
+    testSinglePeriodCouponReplication(start, end, spread, RateAveraging::Compound, 3);
+    testSinglePeriodCouponReplication(start, end, spread, RateAveraging::Simple, 4);
```

Comment:
I would also check the common case fixingDays = 2 (unless there's already another test case that does).

Reply:
Sure, good to add it for the sake of completeness.